code/modules/modular_computers/computers/item/tablet_presets.dm

```diff
@@ -19,6 +22,12 @@
 	install_component(new /obj/item/weapon/computer_hardware/network_card)
 	install_component(new /obj/item/weapon/computer_hardware/card_slot)
 	install_component(new /obj/item/weapon/computer_hardware/printer/mini)
+	install_programs()
+
+/obj/item/device/modular_computer/tablet/preset/advanced/engi/install_programs()
+	var/obj/item/weapon/computer_hardware/hard_drive/hard_drive = all_components[MC_HDD]
+	hard_drive.store_file(new/datum/computer_file/program/alarm_monitor())
+	hard_drive.store_file(new/datum/computer_file/program/supermatter_monitor())
```

Comment:
Please do, I believe tablets are eventually supposed to be a: build your own pda sort of deal